public/resources/remix/style.less

```diff
@@ -21,6 +21,7 @@
 }
 
 .details-bar {
+  display: none !important;
```

Comment:
This should not be necessary. Instead, the associated DOM element should probably simply not even get generated (if the env vars say it shouldn't be)?

Reply:
yeah I forgot that I need to do this based on an env var